Replicator/CivetWebSocket.cc

```diff
@@ -166,7 +166,7 @@ namespace litecore { namespace websocket {
             Log("CivetWebSocket connecting to <%.*s>...", SPLAT(_address.url()));
 
             stringstream extraHeaders;
-            for (Dict::iterator header(_options[kC4ReplicatorOptionExtraHeaders].asDict());
+                for (Dict::iterator header(_options[kC4ReplicatorOptionExtraHeaders].asDict());
```

Comment:
Indent level is wrong -- make sure there are no tabs, please. (Same for the two diffs below.) Actually looks like there are no changes aside from the indents, so you could remove the diffs.